vm/qemu/qemu.go

```diff
@@ -142,7 +143,7 @@ var archConfigs = map[string]*archConfig{
 	"netbsd/amd64": {
 		Qemu:      "qemu-system-x86_64",
 		TargetDir: "/",
-		QemuArgs:  "-enable-kvm",
+		QemuArgs:  "-machine type=pc",
```

Comment:
Execution will be very slow without kvm. Are you testing on a machine without kvm support? But the previous version of this change did not include this, so it somehow worked with kvm.
The general recommendation is to pass custom "qemu_args" to disable kvm if necessary. But this won't work for pkg/build because user can't affect "qemu_args" used by pkg/build.
So do you need this? At the very least let's move this to pkg/build.

Reply:
ping